stories/Dropdown.stories.tsx

```diff
@@ -71,22 +82,22 @@ interface Icon {
 }
 
 const iconData: DropdownOption<Icon>[] = [
-  { value: '1', displayName: 'app' },
-  { value: '2', displayName: 'bot' },
-  { value: '9', displayName: 'hide' },
-  { value: '10', displayName: 'link' },
-  { value: '3', displayName: 'adjust' },
-  { value: '4', displayName: 'archive' },
-  { value: '5', displayName: 'cashtag' },
-  { value: '6', displayName: 'emoticon' },
-  { value: '7', displayName: 'following' },
-  { value: '8', displayName: 'flags' }
+  { value: '1', displayName: 'app', label: 'app' },
```

Comment:
isn't working anymore without the label? The displayName is supposed to be the label.

Reply:
Because I wanted to remove the `filterFunction` prop in the "custom optionRenderer" example. But the default filter depends on the label attribute.
![image](https://user-images.githubusercontent.com/56824367/146237615-12768394-8306-4dc9-99d3-b2e44e5c93dc.png)